tests/integration/workflows/helpers/calls.py

```diff
@@ -167,6 +167,28 @@ def generate_record():
     return json_data
 
 
+def insert_citing_record():
+    """Provide record fixture."""
+    json_data = json.loads(pkg_resources.resource_string(
+        __name__,
+        os.path.join(
+            '../fixtures',
+            'oai_arxiv_citing_record.json'
+        )
+    ))
+
+    categories = {'core': [], 'non-core': []}
+    for eprint in json_data.get('arxiv_eprints', []):
+        categories['core'].extend(eprint.get('categories', []))
+
+    if 'preprint_date' in json_data:
+        json_data['preprint_date'] = datetime.date.today().isoformat()
+
+    assert categories
```

Comment:
what is this for? you are defining `categories` to be non-empty. If you want to assert that the category is in the record, you should assert that.

Reply:
I think this is a very hacky way to get the task done in the first place (to make it auto-accepted). I am  not exactly sure why this was done in the first place in https://github.com/inspirehep/inspire-next/blob/ac4fdbbf50b03b55dd5f4983600ec65cca4879d8/tests/integration/workflows/helpers/calls.py#L170
where I took the inspiration from.